recipe/meta.yaml

```diff
@@ -5,14 +5,12 @@ package:
     version: {{ version }}
 
 source:
-    fn: protobuf-{{ version }}.tar.gz
-    url: https://pypi.python.org/packages/source/p/protobuf/protobuf-{{ version }}.tar.gz
-    md5: f0d3bd2394345a9af4a277cd0302ae83
+    fn: protobuf-{{ version }}.post2.tar.gz
+    url: https://pypi.python.org/packages/source/p/protobuf/protobuf-{{ version }}.post2.tar.gz
```

Comment:
I think the `.post2` should go in the declaration of the `version` variable at the top of the file.